### ISSUE TITLE
River of News

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kiniaru/mobile-rss-reader#5]
```
User should be able to have a single view for all feeds.

Anonymous comment from user:
"...
Also, you should have a “River of News” view for your feed reader.
Sometimes I just want to read the new items regardless of what feed they
are from. Essentially this would be a view where the latest entries would
be ordered based on time, regardless of which feed they originate from."

```

Original issue reported on code.google.com by `tlaukka...@gmail.com` on 23 Sep 2006 at 6:12

**Comment by GoogleCodeExporter** [comment]
```
This is in release 1.10.5.  After getting to unread items, you can select Date 
sort 
items from menu.
```

Original comment by `ibunto...@gmail.com` on 18 Sep 2007 at 11:20
- Added labels: **Milestone-Release1.10.5**
- Removed labels: **Milestone-Release1.4**

**Comment by GoogleCodeExporter** [comment]
```
River of news i great. Would be good if each title is prefixed with the feed it 
is 
from. E.g

[CNN Technology] Aliens found on Mars

THis helps to put the headline into context. This would be further helped with 
introduction of font control, so that small fonts will ensure this feature will 
not 
overcrowd the screen (esp in a portrait mode)

Thanks!

David
davidwcy@gmail.com
```

Original comment by `david...@gmail.com` on 4 Nov 2008 at 12:55

**Comment by GoogleCodeExporter** [comment]
```
Hi David,

I've done some of this in the dev 2 version which has font size for lists and 
wraparound also for lists and feed title without braces.  I will put in the 
braces 
in dev 3 release.

Regards,

Irving
```

Original comment by `ibunto...@gmail.com` on 22 Dec 2008 at 1:35

**Comment by GoogleCodeExporter** [comment]
```
This is fixed on version 1.11.2.  Now has font size control and can change sort 
from
ascending to descinding.  Also, can change from feed order to descending feed 
order.
 Now, headers (items) has the same functionality as "River of News".
```

Original comment by `ibunto...@gmail.com` on 14 May 2009 at 5:16
- Changed state: **Fixed**
- Added labels: **Milestone-Release1.11.2**
- Removed labels: **Milestone-Release1.10.5**
